trieste/models/gpflow/models.py

```diff
@@ -112,34 +114,53 @@ def covariance_between_points(
 
         .. math:: \Sigma_{12} = K_{12} - K_{x1}(K_{xx} + \sigma^2 I)^{-1}K_{x2}
 
-        :param query_points_1: Set of query points with shape [N, D]
+        :param query_points_1: Set of query points with shape [..., N, D]
```

Comment:
we should have a test with these two cases of query_points_1

Reply:
also, to a general user N, D, M and L might be unclear what it stands for - perhaps putting abbreviation definitions in the class docstring?

Reply:
L is the number of latent GPs. So for GPR it's the number of output dims.